packages/perseus-editor/src/__stories__/editor-page-util.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+    Renderer,
+    type APIOptions,
+    type DeviceType,
+    type Hint,
+    type PerseusAnswerArea,
+    type PerseusRenderer,
+} from "@khanacademy/perseus";
+import Button from "@khanacademy/wonder-blocks-button";
+import {View} from "@khanacademy/wonder-blocks-core";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import {color, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+import xIcon from "@phosphor-icons/core/regular/x.svg";
+import {action} from "@storybook/addon-actions";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+// eslint-disable-next-line import/no-relative-packages
+import {mockStrings} from "../../../perseus/src/strings";
+import EditorPage from "../editor-page";
+
+import {flags} from "./flags-for-api-options";
+
+type Props = {
+    apiOptions?: APIOptions;
+    question?: PerseusRenderer;
+    hints?: ReadonlyArray<Hint>;
+};
+
+const onChangeAction = action("onChange");
+
+function EditorPageUtil(props: Props) {
```

Comment:
The name `EditorPageUtil` isn't particularly intuitive / self-explanatory to me. Maybe this component could be named something like `EditorPageWithStorybookPreview`?